### PR TITLE
fix(main/libslirp): Patch libslirp to use Termux's own resolv.conf

### PR DESCRIPTION
**Pull request by kyufie**
This patch addresses DNS-related failures due to the absence of `/etc/resolv.conf`.
Consequently, this patch also fixes DNS-related failures in QEMU VMs that use nameservers provided by QEMU's own DHCP server.

### CODE REVIEW

**Review by TomJo2000 — APPROVED** [review]
Thank you for the contribution.
Looks all good to me, just waiting for the CI run to finish before merging.

**Comment by TomJo2000** [comment]
Thank you again for your contribution.

**Comment by kyufie** [comment]
You're welcome :)

**Pull request merged**
